internal/gcsx/file_cache_reader.go

```diff
@@ -33,6 +33,12 @@ import (
 	"github.com/jacobsa/fuse/fuseops"
 )
 
+const (
+	// ReadOp ("readOp") is the value used in read context to store pointer to the read operation.
+	ReadOp = "readOp"
+	MiB    = 1 << 20
```

Comment:
Instead of keeping the constants here, you can create a constants.go file in the gcsx package, which will be used by all the readers.

Reply:
I feel currently we only have two constants so we should not create separate file. If in future it will require then I'll create at that time.